packages/tv/.eslintrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
Can we use the same `.eslintcrc.js` we're using in `/mobile`? Just to keep it consistent 👍